server/dto/service_quality_response.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2017 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package dto
+
+// ServiceQualityResponse represents response from the quality oracle service
+type ServiceQualityResponse struct {
+	Connects []QualityConnects `json:"connects"`
+}
+
+// QualityConnects represents a single proposal connection quality description
+type QualityConnects struct {
+	Proposal     QualityProposal `json:"proposal"`
+	CountAll     int             `json:"countAll"`
```

Comment:
I dont think we need to map MORQA response to local DTOs, because new field adding will be complex.
Lets just map MORQA response by `proposalID` and proxy it in Tequilapi

Reply:
Now it matches MORQA responses and just proxy it.